### DISCUSSION TITLE
Who uses AGE?

### DISCUSSION THREAD

**Discussion by JoshInnis** [apache/age#226] [General] [answered]
Hello, we were wondering if anybody wanted to share how they are using Apache AGE?

**Comment by jayveeAtWork** [comment, marked as answer]
Use case = dependencies between process runs, visualized as a graph. 
Currently doing that in regular SQL with recursive queries and actually working fine (if not very readable). We're interested in whether a graph database might make lookups easier to understand, and whether it will provide additional functionality benefitting our use case. Though at this point, we're just sniffing around for a potential POC later this year.
